docs-src/go/how-to-update-a-schedule-in-go.md

```diff
@@ -3,6 +3,10 @@ id: how-to-update-a-schedule-in-go
 title: How to update a Schedule in Go
 sidebar_label: Update Schedule
 description: To update a Schedule in Go, use `updateSchedule()` on the ScheduleHandle.
+tags:
+  - schedule
+  
```

Comment:
looks like search and replace left an empty newline?

lets fix this - just in case.